packages/flutter/lib/src/material/selectable_text.dart

```diff
@@ -406,6 +408,9 @@ class SelectableText extends StatefulWidget {
   /// {@macro flutter.widgets.editableText.scrollPhysics}
   final ScrollPhysics scrollPhysics;
 
+  /// {@macro flutter.painting.textPainter.textHeightBehavior}
```

Comment:
Did you define this one and forgot to copy it over or does it already exist ?

Reply:
Did you mean `{@macro flutter.dart:ui.textHeightBehavior}`, which is what we are using as the macro in `text_painter.dart`

Reply:
Thanks for the spot, I updated all the macro references to `{@macro flutter.dart:ui.textHeightBehavior}`